azurelinuxagent/ga/agent_update_handler.py

```diff
@@ -48,6 +48,7 @@ class AgentUpdateHandler(object):
                     version: it will have what version to update
                     isVersionFromRSM: True if the version is from RSM deployment.
                     isVMEnabledForRSMUpgrades: True if the VM is enabled for RSM upgrades.
+                    fromVersion: This property specifies the version to update from. It is populated only for downgrade requests and subsequent goal states thereafter, until an upgrade request.
```

Comment:
Once an update request comes, what value will this property be in the goal state? None or not provided?

Reply:
only present in downgrade requests